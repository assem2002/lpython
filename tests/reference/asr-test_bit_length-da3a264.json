{
    "basename": "asr-test_bit_length-da3a264",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_bit_length.py",
    "infile_hash": "877db12847eccaf31564c6a969e7703f16c9e03bc9c326e72c74f1e0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_bit_length-da3a264.stderr",
    "stderr_hash": "588be2ebf4b260ff90313b58763bcb37eceab1d0e9cae576851f7182",
    "returncode": 2
}