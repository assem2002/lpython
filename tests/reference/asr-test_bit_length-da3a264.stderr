semantic error: int.bit_length() takes no arguments
 --> tests/errors/test_bit_length.py:4:11
  |
4 |     print(x.bit_length(23))
  |           ^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
